cx_Freeze/freezer.py

```diff
@@ -279,6 +279,18 @@ def _freeze_executable(self, exe: Executable) -> None:
             copy_dependent_files=False,
             include_mode=True,
         )
+
+        # copy a file with a the cx_freeze license into frozen application
+        respath = get_resource_file_path(
+            "freeze_license", "cx_frozen_license", ".txt"
+        )
+        self._copy_file(
+            respath.absolute(),
+            self.targetdir / "cx_frozen_application_license.txt",
```

Comment:
cx_frozen_application_license -> frozen_application_license or cx_freeze_application_license
cx_freeze/cx_Freeze is the package name

Reply:
I'll go with frozen_application_license